* grobian@gentoo.org: open the files in the same order the Makefile
                      depends on them, such that we don't trigger a
                      remake all the time, see:
                      http://bugs.gentoo.org/show_bug.cgi?id=200662

--- info/makedoc.c
+++ info/makedoc.c
@@ -145,8 +145,8 @@
     }
 
   funs_stream = must_fopen (funs_filename, "w");
-  doc_stream = must_fopen (doc_filename, "w");
   key_stream = must_fopen (key_filename, "w");
+  doc_stream = must_fopen (doc_filename, "w");
 
   fprintf (funs_stream,
       "/* %s -- Generated declarations for Info commands. */\n\n\
@@ -226,8 +226,8 @@
   fprintf (funs_stream, "\n#define A_NCOMMANDS %u\n", next_func_key());
 
   fclose (funs_stream);
-  fclose (doc_stream);
   fclose (key_stream);
+  fclose (doc_stream);
 
   if (tags_only)
     maybe_dump_tags (stdout);
